File: pinyougou-parent/pinyougou-shop-web/src/main/webapp/plugins/angularjs/service/seckillGoodsService.js
app.service('seckillGoodsService', function ($http) {

    this.save = function (entity) {
        return $http.post('../seckillGoods/save.do', entity);
    };

    this.findByPage = function (page, size, searchEntity) {
        return $http.post('../seckillGoods/findByPage.do?page=' + page + '&size=' + size, searchEntity);
    };

    this.findById = function (id) {
        return $http.post('../seckillGoods/findById.do?id=' + id);
    };
});

File: pinyougou-parent/pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;
import entity.PageResult;

import java.util.List;

public interface SeckillGoodsService {

    PageResult<TbSeckillGoods> findByPage(TbSeckillGoods seckillGoods, int pageNum, int pageSize) throws Exception;

    TbSeckillGoods findById(Long id) throws Exception;

    void save(TbSeckillGoods seckillGoods) throws Exception;

    List<TbSeckillGoods> findList() throws Exception;

    TbSeckillGoods findByIdFromRedis(Long id) throws Exception;
}